testing/tests/c_maths.py

```diff
@@ -0,0 +1,16 @@
+"""
+Tests on the c_maths project
+"""
+
+from testing.base import ExhaleTestCase
```

Comment:
So for python 2 it's unable to import `testing.base`.  I tried to import `exhale` just above it but the CI logs are inaccessible right now.  I'm leaning toward ditching buildout, if I can't test py2 it's a no-go (since many people still use that on RTD).

Reply:
it has nothing to see with buildout really. buildout does not care if you're using python 2 or 3, it just generates a script and sets sys.path according to the packages it's told to include.

Reply:
afterthought: did you add the missing ``__init__.py`` files before testing with python 2?? Because without them python 2 is lost.

Reply:
yeah i added some `__init__.py` and various other things, the python2 importing now works!